src/helix-ui/mixins/Revealable.js

```diff
@@ -42,6 +42,11 @@ export const Revealable = (superclass) => {
             if (attr === 'open') {
                 let isOpen = (newVal !== null);
                 this.setAttribute('aria-hidden', !isOpen);
+                // TODO: Emit events only when setting the 'open' property.
+                // - emit 'open' when open=true
+                // - emit 'close' when open=false
+                // - allow user to cancel the event, if desired
+                // - ensure that Positionable is updated accordingly
```

Comment:
comment added, because this change may affect Positionable functionality

- [ ] (@CITguy) create JIRA to capture refactoring "open" and "close" events to enable user cancelation